src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/factory/transformNodeFuntions/transformNodeFunctions.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import React from 'react';
+
+export const transformTrackerNode = (node: Object, index: number, nodeToElementFn: (...args : Array<any>) => any) => {
+    if (node.name === 'input') {
+        const htmlElementId = node.attribs?.attributeid;
+
+        if (htmlElementId) {
+            const inputElement = nodeToElementFn(node, index);
+
+            const style = inputElement.props?.style;
+            const className = inputElement.props?.className;
+
+            const customFormElementProps = {
+                id: htmlElementId,
+                style,
+                className,
+            };
+
+            return React.createElement(
+                'FormField', {
+                    customFormElementProps,
+                    id: htmlElementId,
+                },
+            );
+        }
+        return null;
```

Comment:
Do we need this? To be clear: If we have an input node without an attributeId I would prefer that it was rendered to the DOM without any transformation.

Reply:
Noted, I've removed it.